zds/pages/tests.py

```diff
@@ -65,8 +65,16 @@ def test_url_association(self):
         self.assertEqual(result.status_code, 200)
 
     def test_subscribe_association(self):
-        """To test the "subscription to the association" form."""
+        """
+        To test the "subscription to the association" form.
+        """
+        forum_category = CategoryFactory(position=1)
+        forum = ForumFactory(category=forum_category, position_in_category=1)
 
+        # override the settings to avoid 404 if forum do not exist
```

Comment:
```diff
-        # override the settings to avoid 404 if forum do not exist
+        # overrides the settings to avoid 404 if forum does not exist
```